assets/dev/scss/frontend/_container.scss

```
@@ -101,8 +101,8 @@
 
 	&-boxed.e-grid {
 		justify-items: initial;
-		grid-template-columns: initial;
-		grid-template-rows: initial;
+		grid-template-columns: 1fr;
```

```suggestion
		grid-template-columns: repeat(3, 1fr);
```
Just making sure? This is the default according to the spec.

---

@hein-obox We also have a variable for these in the PHP? `--e-con-grid-template-rows` and `--e-con-grid-template-columns`

---

@louiswol94 

My change only applies to the wrapper container (&-boxed.e-grid), in case there is an inner container. The wrapper needs to stay full width. This doesn't apply to the display of the content inside the container itself.

I hope I am making sense.